src/test/java/uk/gov/hmcts/reform/bulkscan/orchestrator/model/ccd/mappers/ExceptionRecordMapperTest.java

```diff
@@ -184,6 +202,17 @@ public void mapEnvelope_sets_display_case_reference_fields_to_yes_when_envelope_
         assertThat(exceptionRecord.showEnvelopeLegacyCaseReference).isEqualTo("Yes");
     }
 
+    @Test
+    public void mapEnvelope_sets_surname_null_when_no_surname_data_in_ocr() {
+        //given
+        Envelope envelope = envelope("CASE_123", "LEGACY_CASE_123", Classification.SUPPLEMENTARY_EVIDENCE_WITH_OCR);
+        // when
+        ExceptionRecord exceptionRecord = mapper.mapEnvelope(envelope);
+
+        // then
+        assertThat(exceptionRecord.surname).isNull();
+    }
```

Comment:
missing test for duplicated surnames in OCR

Reply:
added